src/main/java/org/fundacionjala/coding/melvi/MovieRefactor/ReportGenFullFormat.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.coding.melvi.MovieRefactor;
+
+import java.util.Enumeration;
+
+/**
+ *
+ */
+public class ReportGenFullFormat extends ReportGenAbstract {
+    /**
+     * @param rentals          rentyals to print
+     * @param customerName     custmoer name
+     * @param priceCalculation price calculation method
+     * @return string
+     */
+    public String print(final Enumeration rentals, final String customerName,
+                        final IPriceCalculation priceCalculation) {
+        StringBuilder sb = new StringBuilder();
+        double totalAmount = 0;
+        int frequentRenterPoints = 0;
+        sb.append("Rental Record for " + customerName + "\n");
+        while (rentals.hasMoreElements()) {
+            Rental each = (Rental) rentals.nextElement();
+            double thisAmount = priceCalculation.getPrice(each.getMovie(), each.getDaysRented());
+
+            // add frequent renter points
+            frequentRenterPoints++;
+            frequentRenterPoints += each.getMovie().getBonusPoints(each.getDaysRented());
+            //show figures for this rental
+            sb.append("\t");
+            sb.append(each.getMovie().getTitle());
+            sb.append("\t");
+            sb.append(String.valueOf(thisAmount));
+            sb.append("\n");
+            totalAmount += thisAmount;
+        }
+        //add footer lines
+        sb.append("Amount owed is " + String.valueOf(totalAmount));
+        sb.append("\n");
+        sb.append("You earned " + String.valueOf(frequentRenterPoints));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Directly append the argument of String.valueOf(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1153)